### DISCUSSION TITLE
Original OS

### DISCUSSION THREAD

**Discussion by lawrencejoy** [iiab/iiab#3764] [General]
Hi is there a dual network option IE IIAB on a pi 4 using the internal wifi as hotspot and then a WIFI dongle to allow access to the original PI OS  , to allow other programs such as software radio etc ?

thank you

**Comment by tim-moody** [comment]
Let's start with what you mean by original Pi OS. IIAB does not change the OS. It does change networking in the sense that it dedicates the internal Wi-Fi on a pi 4 to be a hotspot. But it also allows that internal Wi-Fi to be dual hosted so that it can serve as both a hotspot and a means of connecting to an upstream router.

However if you want to add a Wi-Fi dongle I think that will  also be available for an upstream connection, though you will have to configure the SSID and password outside of IIAB. You can then install anything  you like to use that device.
